src/BlazorUI/Bit.BlazorUI/Components/Notifications/Badge/BitBadge.scss

```
@@ -23,24 +26,135 @@
+    white-space: nowrap;
     gap: spacing(0.5);
     position: absolute;
     align-items: center;
-    pointer-events: all;
+    justify-content: center;
+    pointer-events: none;
     display: inline-flex;
     box-sizing: border-box;
     width: var(--bit-bdg-width);
     height: var(--bit-bdg-height);
+    // A counter is only ever as wide as it is tall until a second digit pushes it wider, which is what
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)